Sources/App/Views/Sections/Editor/SourceTree/SourceTreeViewModel.swift

```diff
@@ -42,8 +56,8 @@ final class SourceTreeViewModel: ObservableObject {
   ///
   /// This instantiation will fail if the workspace path value has not been set yet.
   /// - Throws: `MockaError.missingWorkspacePathValue` if `path` is `nil`.
-  init() throws {
-    try refreshContent()
+  init(editorEnvironment: EditorEnvironment) {
+    self.editorEnvironment = editorEnvironment
```

Comment:
Incomplete documentation

```suggestion
  init(editorSectionEnvironment: EditorSectionEnvironment) {
    self.editorSectionEnvironment = editorSectionEnvironment
```